src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1765,6 +1785,7 @@ private function registerMessengerConfiguration(array $config, ContainerBuilder
             $container->removeDefinition('messenger.transport.symfony_serializer');
             $container->removeDefinition('messenger.transport.amqp.factory');
             $container->removeDefinition('messenger.transport.redis.factory');
+            $container->removeDefinition('messenger.transport.sqs.factory');
```

Comment:
This line should not be part of 4.4, right?

Reply:
indeed, fixed